config/initializers/active_metrics.rb

```diff
@@ -0,0 +1 @@
+ActiveMetrics::Base.establish_connection(:adapter => "miq_postgres", :database => "metrics_spike")
```

Comment:
O_o Good catch, it absolutely matters does it not? This should absolutely fall on its face without a change.